content/actions/guides/publishing-java-packages-with-gradle.md

```diff
@@ -144,7 +142,7 @@ publishing {
 ```
 {% endraw %}
 
-With this configuration, you can create a workflow that publishes your package to the Maven Central Repository by running the `gradle publish` command.
+With this configuration, you can create a workflow that publishes your package to GitHub Packages by running the `gradle publish` command.
```

Comment:
```suggestion
With this configuration, you can create a workflow that publishes your package to {% data variables.product.prodname_registry %} by running the `gradle publish` command.
```

Reply:
Ezzel a konfigurációval a `gradle publish` parancs futtatásával létrehozhat egy munkafolyamatot, amely a Maven Central Repository-ban és a {% data variables.product.prodname_registry %}-ban is közzéteszi a csomagot